plugins/inputs/openntpd/sample.conf

```diff
@@ -7,4 +7,4 @@
   # binary = "/usr/sbin/ntpctl"
 
   ## Maximum time the ntpctl binary is allowed to run.
-  # timeout = "5ms"
+  # timeout = "1s"
```

Comment:
The actual default is `5s` in the code, so please change the value accordingly. The commented value should correspond to the actually used default value...